### PR TITLE
Some changes

### PR DESCRIPTION
**Pull request by gestap**
Added `dist` to gitignore, a new package for gh-page. Removed all in the readme and add a link to the guideline.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ node_modules
 .DS_Store
 yarn.lock
 package-lock.json
+dist
```
Comment (roman-mate) [inline .gitignore]: @ivadimko I remember you said we can't add dist to gitignore. Do you remember the reason?

Comment (ivadimko) [reply .gitignore]: With `dist` under `.gitignore` we can't deploy it to GH pages. Our deploy script simply push `dist` folder so it should be committed.

Comment (gestap) [reply .gitignore]: I solve this problem whit plugin `gh-pages`

Comment (roman-mate) [reply .gitignore]: @ivadimko I guess in that case we can move forward.

**Pull request merged**